edgexfoundry/device-mqtt-go
# refactor!: Change configuration and device toml files to yaml

BREAKING CHANGE: Configuration and device files now use yaml

closes: #552







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/device-mqtt-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions
1. run `make test`
2. start edgex with no-secty mqtt-broker
3. `run ./device-mqtt -cp `
4. verify that configuration and device profiles has loaded
5.  `run ./device-mqtt -cp -o`
6.  verify that configuration and device profiles has loaded

## New Dependency Instructions (If applicable)
